src/lib/components/02-components/cta/cta.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import CTA from './cta';
+
+const meta = {
+  title: 'Components/CTA',
+  component: CTA,
+  argTypes: {
```

Comment:
@robherba I think we can get rid of `argTypes` and let storybook infers the props

Reply:
I've removed it, but if the user enters an unexpected topic, the component breaks. 

I've added extra validation to prevent this, but this situation won't occur in the nextjs app because the `theme` field is already being validated with TypeScript.

Reply:
<img width="1314" alt="Captura de pantalla 2024-05-03 a la(s) 2 20 13 p  m" src="https://github.com/fourkitchens/forcepoint-shared-components/assets/9342274/accb29dc-fc65-4eb6-bd53-bb93295a7cd9">

Reply:
Additionally, future developers will need to consult the component's code or documentation to determine which themes are supported.

Reply:
https://github.com/fourkitchens/forcepoint-shared-components/assets/9342274/99492772-2a64-49e7-ac8a-0eaeeb916e2e